packages/ti_crowdstrike/data_stream/intel/elasticsearch/ingest_pipeline/default.yml

```
@@ -347,20 +336,41 @@ processors:
       target_field: ti_crowdstrike.intel.value
       ignore_missing: true
   - set:
-      field: file.hash.sha256
-      tag: set_file_hash_sha256
+      field: threat.indicator.file.hash.sha256
+      tag: set_threat_indicator_file_hash_sha256
       value: '{{{ti_crowdstrike.intel.value}}}'
       if: ctx.ti_crowdstrike?.intel?.type != null && ctx.ti_crowdstrike.intel.type.contains('hash_sha256') && ctx.ti_crowdstrike?.intel?.value != null
   - set:
-      field: file.hash.md5
-      tag: set_file_hash_md5
+      field: threat.indicator.file.hash.sha1
+      tag: set_threat_indicator_file_hash_sha1
+      value: '{{{ti_crowdstrike.intel.value}}}'
+      if: ctx.ti_crowdstrike?.intel?.type != null && ctx.ti_crowdstrike.intel.type.contains('hash_sha1') && ctx.ti_crowdstrike?.intel?.value != null
```

```suggestion
      if: ctx.ti_crowdstrike?.intel?.type != null && ctx.ti_crowdstrike.intel.value != null && ctx.ti_crowdstrike.intel.type.contains('hash_sha1')
```
Removes unnecessary null-safe operators and puts cheaper checks first; similar changes below.